### PR TITLE
chore: Update README and snyk-monitor README

### PR DESCRIPTION
**Pull request by agranado2k**
Reorganise the steps to make them clear to the user.

- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

This PR is updating the README and snyk-monitor/README,  reorganising the steps when creating the snyk-monitor secret.

### More information

- [Jira ticket RUN-755](https://snyksec.atlassian.net/browse/RUN-755)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,26 +54,9 @@ Both of these items must be provided from a Kubernetes secret. The secret must b
...
-The Snyk Integration ID is a UUID and looks similar to the following:
-```
-abcd1234-abcd-1234-abcd-1234abcd1234
-```
-The Snyk Integration ID is used in the `--from-literal=integrationId=` parameter in the next step.
-
-3. Finally, create the secret in Kubernetes by running the following command:
-```shell
-kubectl create secret generic snyk-monitor -n snyk-monitor --from-file=./dockercfg.json --from-literal=integrationId=abcd1234-abcd-1234-abcd-1234abcd1234
-```
```
Comment (ivanstanev) [inline README.md:60]: Could we preserve this command and put it under step 3? Right now there isn't a very clear instruction how to use it! 🙏

File: README.md [hunk 2]
```diff
@@ -41,26 +54,9 @@ Both of these items must be provided from a Kubernetes secret. The secret must b
...
-
 Note that the secret _must_ be namespaced, and the namespace (which we configured earlier) is called _snyk-monitor_.
```
Comment (ivanstanev) [inline README.md:57]: This line feels like it is now too further down at the bottom, could we move it somewhere around step 2, maybe before "Continue to YAML files installation instructions below."?
Or alternatively just delete it, I don't see a similar instruction in our public docs, which should be the most accurate

File: README.md (hunk 1 above)
Comment (agranado2k) [reply README.md:60]: Good catch! I'm including it.

File: README.md (hunk 2 above)
Comment (agranado2k) [reply README.md:57]: I'm deleting! :)

**Pull request merged**